src/DDTrace/Configuration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace DDTrace;
+
+use DDTrace\Configuration\AbstractConfiguration;
+
+/**
+ * DDTrace global configuration object.
+ */
+class Configuration extends AbstractConfiguration
+{
+    /**
+     * Whether or not distributed tracing is enabled globally.
+     *
+     * @return bool
+     */
+    public function isDistributedTracingEnabled()
+    {
+        return $this->registry->boolValue('distributed.tracing', true);
```

Comment:
Can you use `$this->boolValue(...`  here ?

Reply:
eheheh very good catch!!! 👍 thanks